### PR TITLE
Create initial project configuration.

### PR DESCRIPTION
**Pull request by nicolasstucki**

### CODE REVIEW

**Comment by nicolasstucki** [comment]
Review by @sjrd

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+script:
+  - sbt ++$TRAVIS_SCALA_VERSION root/scalastyle testSuiteJVM/scalastyle testSuiteJVM/test:scalastyle testSuite/scalastyle testSuite/test:scalastyle
+  - sbt ++$TRAVIS_SCALA_VERSION testSuiteJVM/test
+  - sbt ++$TRAVIS_SCALA_VERSION testSuite/test
+  - sbt ++$TRAVIS_SCALA_VERSION 'set scalaJSUseRhino in Global := false' testSuite/test
+  - sbt ++$TRAVIS_SCALA_VERSION 'set scalaJSUseRhino in Global := false' 'set scalaJSStage in Global := FullOptStage' testSuite/test
+  - sbt ++$TRAVIS_SCALA_VERSION publishLocal
+scala:
+  - 2.10.6
+  - 2.11.7
```
Comment (sjrd) [inline .travis.yml]: 2.11.8?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# Contributing guidelines
+
+See the [contributing guidelines of Scala.js core](https://github.com/scala-js/scala-js/blob/master/CONTRIBUTING.md).
+The same guidelines apply to this repository.
+
+## Very important notice
+
+`scalajs-java-time` contains a reimplementation of part of the JDK in Scala.js itself.
```
Comment (sjrd) [inline CONTRIBUTING.md]: Copy/paste leftover.

File: README.md [hunk 3]
```diff
@@ -1,2 +1,25 @@
-# scala-js-java-logging
-Scala.js implementation for java.logging
+# scalajs-java-logging
+
+[![Build Status](https://travis-ci.org/scala-js/scala-js-java-logging.svg?branch=master)](https://travis-ci.org/scala-js/scala-js-java-logging)
+[![Scala.js](https://www.scala-js.org/assets/badges/scalajs-0.6.6.svg)](https://www.scala-js.org/)
```
Comment (sjrd) [inline README.md]: You should use the 0.6.8 badge.

File: build.sbt [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+    pomIncludeRepository := { _ => false }
+  )
+
+lazy val testSuite = CrossProject(
+  jvmId = "testSuiteJVM",
+  jsId = "testSuite",
+  base = file("testSuite"),
+  crossType = CrossType.Full
+).
+  jsConfigure(_ .enablePlugins(ScalaJSJUnitPlugin)).
```
Comment (sjrd) [inline build.sbt]: Spurious space between `_` and `.`

File: project/build.sbt [hunk 5]
```diff
@@ -0,0 +1,3 @@
+addSbtPlugin("org.scala-js" % "sbt-scalajs" % "0.6.8")
```
Comment (sjrd) [inline project/build.sbt]: You might as well use 0.6.9.

File: testSuite/js/src/test/scala/org/scalajs/testsuite/utils/Platform.scala [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+
+  final val executingInJVMOnJDK6 = false
+
+  final val executingInJVMOnJDK7OrLower = false
+
+  // Members that are only accessible from testSuite/js
+  // (i.e. do no link on the JVM).
+
+  def areTypedArraysSupported: Boolean =
+    runtime.Bits.areTypedArraysSupported
```
Comment (sjrd) [inline testSuite/js/src/test/scala/org/scalajs/testsuite/utils/Platform.scala]: I guess `areTypedArraysSupported` will not be needed in the context of `java.util.logging`

**Comment by sjrd** [comment]
That's all.

File: .gitignore [hunk 7]
```diff
@@ -0,0 +1,18 @@
...
+target/
+lib_managed/
+src_managed/
+project/boot/
+project/plugins/project/
+
+# Scala-IDE specific
+.scala_dependencies
+.worksheet
+
```
Comment (sjrd) [inline .gitignore]: This `.gitignore` is big. Normally it should only contain

```
target/
```

**Comment by sjrd** [comment]
That's all.

**Comment by sjrd** [comment]
LGTM

**Pull request merged**